dbfit-java/snowflake/src/main/java/dbfit/environment/SnowflakeProcedureParametersParser.java

```diff
@@ -0,0 +1,69 @@
+package dbfit.environment;
+
+import dbfit.util.Direction;
+import dbfit.util.ParamDescriptor;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import static dbfit.util.Direction.*;
+
+public class SnowflakeProcedureParametersParser {
+    private Pattern returnPattern = Pattern.compile("(?<type>\\w+)" + "(\\W)|$");
+
+    private Pattern parametersPattern = Pattern.compile(
+        "((?<direction>\\w+)\\s+)?" +
+        "(?<name>\\w+)" + "\\s+" +
+        "(?<type>\\w+)" + "\\s*" +
+        "(?<options>\\([^)]*\\))?\\s*(,|$)");
+
+    private Pattern outerPattern = Pattern.compile(
+            "^\\(*|\\)*$");
+
+    public List<ParamDescriptor> parseParameters(String parametersExpression) {
+        parametersExpression = parametersExpression.replaceAll("^\\(*|\\)*$","");
```

Comment:
Can we reuse the same string of regex via some constant - as we have it on multiple places.